pkg/drainer/key.go

```diff
@@ -0,0 +1,37 @@
+package drainer
+
+import (
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func isCriticalPod(podName string) bool {
```

Comment:
I'm unsure about this function. Don't really like having the critical pods hardcoded.

Is it possible to instead infer that they're critical from their priority class or similar?

Reply:
This comes straight from node-operator https://github.com/giantswarm/node-operator/blob/36ad1d7b5ff3a5c9fea14246c1b5a1f80083e27c/service/controller/key/key.go#L25

I agree with you we could improve it, but I'm already out of time with this release I'd leave it like it is if you don't mind

Reply:
Yeah, don't think it's a blocker. Just something to be mindful of I think :)